### DISCUSSION TITLE
Vlan Tagging?

### DISCUSSION THREAD

**Discussion by calimansi** [gtxaspec/wz_mini_hacks#756] [Q&A]
Hi,

Is there a way to configure wz mini hacks to tag all traffic from a wyze v3 cam with a vlan tag?  Trying to avoid purchasing a smart switch.

**Comment by virmaior** [comment]
it looks busybox (the underlying linux tool system for Wyze V2 and V3s and many embedded systems) has this with a util called `vconfig` (https://www.busybox.net/BusyBox.html ).

**Reply by calimansi** [reply]
Good guidance!!!  I tried vconfig add wlan0 10 and it gives me a vague error.  I believe the error is related to the missing 8021q module. 

Do you know how I can get that easily or is it a compile thing?  Thanks.

**Comment by virmaior** [comment]
unfortunately i don't know ...
